vllm/sequence.py

```diff
@@ -111,7 +111,15 @@ class RequestMetrics:
     model_execute_time: Optional[float] = None
 
 
-class SequenceData:
+class SequenceDataDelta(msgspec.Struct, array_like=True, omit_defaults=True):
+    """Delta SequenceGroupData to send to workers."""
+    new_output_token_ids: List[int]
```

Comment:
Can we add doc for the arg to include info: this is to append to the original list and not to replace it

Reply:
great feedback! fixed